datasets/seq_mnist.py

```diff
@@ -11,8 +11,7 @@
 from PIL import Image
 from torchvision.datasets import MNIST
 
-from datasets.utils.continual_dataset import (ContinualDataset,
-                                              store_masked_loaders)
+from datasets.utils.continual_dataset import ContinualDataset
 from datasets.utils.validation import get_train_val
 from utils.conf import base_path_dataset as base_path
 
```

Comment:
Line 74: `store_masked_loaders( )` function is not defined

Reply:
removed in the final code version on the branch